serving/installing-logging-metrics-traces.md

```diff
@@ -153,6 +151,35 @@ To configure and setup monitoring:
      ```
 
   CTRL+C to exit watch.
+
+## Stackdriver for both logging and metrics in GKE
+
+You can enable Stackdriver in the GCP project by following [this](https://cloud.google.com/monitoring/workspaces/guide#single-project-ws).
+When Knative is deployed to GKE, you can choose to send both logging and metrics to Stackdriver backend. To have a complete monitoring experience, follow these steps:
+
+1. To have [Kubernetes metrics](https://cloud.google.com/monitoring/api/metrics_other#other-kubernetes.io), create a Kubernetes cluster on GKE with [Stackdriver Kubernetes Monitoring](https://cloud.google.com/kubernetes-monitoring/) feature:
+    <pre>
+    gcloud <b>beta</b> container clusters create $CLUSTER_NAME \
+      --zone=$CLUSTER_ZONE \
+      --cluster-version=latest \
+      --machine-type=n1-standard-4 \
+      --enable-autoscaling --min-nodes=1 --max-nodes=10 \
+      --enable-autorepair \
+      <b>--enable-stackdriver-kubernetes \</b>
+      --scopes=service-control,service-management,compute-rw,storage-ro,cloud-platform,logging-write,monitoring-write,pubsub,datastore \
+      --num-nodes=3
+    </pre>
+2. To have Istio mesh metrics including request count, request latencies, use Istio OSS 1.0.4 or Istio addon 1.0.3.
+3. To have Knative system metrics, 
+   1. Update *metrics.backend-destination* to be stackdriver in [config-observability.yaml](https://github.com/knative/serving/blob/388f98a0a4bb6799ecf174aafc768098890e6cba/config/config-observability.yaml#L92).
```

Comment:
Can we include in the release something like "stackdriver-logging&monitoring.yaml" that puts everything together and does this, so we don't make our end users manually patch files and apply them all together?